source/Scenes/Components/ExitMenuOverlay/RefutationButton.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Annex.Graphics.Events;
+using Annex.Scenes.Components;
+using System;
+
+namespace Game.Scenes.Components.ExitMenuOverlay
+{
+    public class RefutationButton : Button
```

Comment:
Is there a need to have two classes for Yes/No? Looking at both of them, they have the same implementation. They're just called differently. Unless, am I missing something?